kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -120,23 +120,23 @@
       </activation>
 
       <properties>
-        <fabric8.mode>openshift</fabric8.mode>
-        <fabric8.build.strategy>docker</fabric8.build.strategy>
+        <jkube.mode>openshift</jkube.mode>
```

Comment:
If run with `-Popenshift` it will deploy on the configured OpenShift cluster? This is fine, but I think we need to also update the docs related to this plugin and any scripts that use this profile. The old plugin won't deploy it directly.